rosdep/base.yaml

```diff
@@ -1682,6 +1682,11 @@ libev-dev:
   fedora: [libev-devel]
   gentoo: [dev-libs/libev]
   ubuntu: [libev-dev]
+libevdev-dev:
+  arch: [libevdev]
+  debian: [libevdev-dev]
+  fedora: [libevdev]
```

Comment:
I believe the equivalent package in fedora is libevdev-devel

https://rpmfind.net/linux/rpm2html/search.php?query=libevdev-devel&submit=Search+...&system=&arch=

Reply:
Hi, I've fixed the changes. libev is different from libevdev and probably should go in the libev rules rather than the libevdev rules